_posts/2015-09-04-ditching-drf-serializers-for-serpy.md

```diff
@@ -0,0 +1,65 @@
+---
+layout: post
+title: Ditching DRF Serializers for Serpy
```

Comment:
might want to just call this out as `Ditching Django Rest Framework Serializers for Serpy`
its better SEO to spell the project out in full